core/performance_test/peakflops/02_kokkos_simple_view.cpp

```diff
@@ -0,0 +1,189 @@
+#include <Kokkos_Core.hpp>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <gtest/gtest.h>
+#include "common.h"
+//#include "rdtsc.h"
+
+typedef Kokkos::View<float[VECLENTH],Kokkos::MemoryTraits<Kokkos::Restrict> > view_type;
+
+/*
```

Comment:
Is this code going to be used? If not remove it.

Reply:
fixed